mmv1/products/vmwareengine/Subnet.yaml

```diff
@@ -0,0 +1,168 @@
+self_link: '{{parent}}/subnets/{{name}}'
+update_mask: true
+create_verb: :PATCH
+update_verb: :PATCH
+skip_delete: true
+references: !ruby/object:Api::Resource::ReferenceLinks
+  api: 'https://cloud.google.com/vmware-engine/docs/reference/rest/v1/projects.locations.privateClouds.subnets'
+description: |
+  Subnet in a private cloud. Either `management` subnets (such as vMotion) that  are read-only,
+  or `userDefined`, which can also be updated.
+async: !ruby/object:Api::OpAsync
+  actions: ['create', 'update']
+  operation: !ruby/object:Api::OpAsync::Operation
+    path: "name"
+    base_url: "{{op_id}}"
+    wait_ms: 1000
+  result: !ruby/object:Api::OpAsync::Result
+    path: "response"
+  status: !ruby/object:Api::OpAsync::Status
+    path: "done"
+    complete: true
+    allowed:
+      - true
+      - false
+  error: !ruby/object:Api::OpAsync::Error
+    path: "error"
+    message: "message"
+  include_project: true
+
+import_format: ["{{%parent}}/subnets/{{name}}"]
+id_format: "{{parent}}/subnets/{{name}}"
+autogen_async: true
+
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: "vmware_engine_subnet_user_defined"
+    skip_test: true   # update tests will take care of read and update. Parent PC creation is expensive and node reservation is required.
+    primary_resource_id: "vmw-engine-subnet"
+    vars:
+      private_cloud_id: "sample-pc"
+      management_cluster_id: "sample-mgmt-cluster"
+      network_id: "pc-nw"
+      subnet_id: "service-1"
+
+parameters:
+  - !ruby/object:Api::Type::String
+    name: "parent"
+    immutable: true
+    required: true
+    url_param_only: true
+    description: |
+      The resource name of the private cloud to create a new subnet in.
+      Resource names are schemeless URIs that follow the conventions in https://cloud.google.com/apis/design/resource_names.
+      For example: projects/my-project/locations/us-west1-a/privateClouds/my-cloud
+
+  - !ruby/object:Api::Type::String
+    name: "name"
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      The ID of the subnet.
+
+properties:
+  - !ruby/object:Api::Type::Time
+    name: 'createTime'
+    output: true
+    description: |
+      Creation time of this resource.
+      A timestamp in RFC3339 UTC "Zulu" format, with nanosecond resolution and
+      up to nine fractional digits. Examples: "2014-10-02T15:01:23Z" and "2014-10-02T15:01:23.045123456Z".
+
+  - !ruby/object:Api::Type::Time
+    name: 'updateTime'
+    output: true
+    description: |
+      Last updated time of this resource.
+      A timestamp in RFC3339 UTC "Zulu" format, with nanosecond resolution and up to nine
+      fractional digits. Examples: "2014-10-02T15:01:23Z" and "2014-10-02T15:01:23.045123456Z".
+
+  - !ruby/object:Api::Type::String
+    name: 'description'
+    output: true
+    description: |
+      User-provided description for this subnet.
```

Comment:
Can use provide `description` to the subnet?

Reply:
Thanks for pointing that out, I have removed the description field.